packages/lib-classifier/src/store/Workflow/SubjectSetStore/SubjectSetStore.js

```diff
@@ -0,0 +1,12 @@
+import { types } from 'mobx-state-tree'
+import SubjectSet from '../SubjectSet'
+import ResourceStore from '../../ResourceStore'
+
+const SubjectSetStore = types
+  .model('SubjectSetStore', {
+    active: types.safeReference(SubjectSet),
+    resources: types.map(SubjectSet),
+    type: types.optional(types.string, 'subject_sets')
```

Comment:
Thinking about this, maybe this should be `types.literal('subject_sets'). I don't know when we would ever want this to be undefined or something else. Out of scope, though, and we would want to make this change across store models.

Reply:
From memory, literals error if they aren't present in the initial snapshot. So, this line allows us to create a new store from an empty object?